tests/files/keystorage/online.key

```diff
@@ -0,0 +1 @@
+ab6854c1929f66157810a40c245de44b@@@@100000@@@@e1e98332e6bbcd3c6f7f0abf7357f43fdba7d014dc60096c3d84df22a77450be@@@@4815a3581e0f4c705698fe732a6ea044@@@@2bfb102cce889a4edfd04b544d51163ac4e45689e76e7e7ee3f047fa41403fb6ebc2707c23778262d5400e7afebd81d46a69a0de09ff6947f4c9dcd339089aa924045263fc884c8501b7ccb5906ce33b72cbd0755f0b8a35ec753a50022cd3ddd592710d269ab94f3390ab5f9fc54e8e3dbf30c0e163fbdaad775b21837f37087d4017cab06b556256a984060fba01d9d4427b74596b8497dfb37edb150fa698d2cb75faedb8b51bfdcf50d3a3be43718d3ad3a5a5913ecf2508753a018ee3f95242da68c2c2f6149fdbd2e88d590fbf47f3a141ba05d24aa05085287430d842d0fb76a4c0c6817fd6ecb354c5bcfee2a9854a578530a28d910183612fac0b2902c5dcf827e04a90820bda4f25bd2631d689faf01c81335b68c3b90213b9a04c5acdc997f505bbd40976b556cca6a3b54f270a7eb6b0d9b6f19be29d6ab8d4a31aedfbb72a4c8cd96ca1cbffe90118ba
```

Comment:
I don't get why this changed.
We have `files/key_storage` already.

Reply:
Not sure why as well.
Will revert this change.